### DISCUSSION TITLE
Highlight/mark destination device

### DISCUSSION THREAD

**Discussion by SN-git-stack** [music-assistant/support#3013] [Feature requests and ideas]
As several destinations can be chosen (eg. Chromecast, AirPlay, etc) it would be great to have an icon in the destination list, which shows from which integration is is provided. As I have several integrations and each device announces itself under the same name, I can't identify which provider is chosen to stream to the destination.

**Comment by OzGav** [comment]
Do you mean the player list like you see here 
![image](https://github.com/user-attachments/assets/fa6ff52e-9cd7-4f0c-94cc-6f322947eaf2)

**Comment by SN-git-stack** [comment]
@OzGav : Exactly, just for some reason you have them, I only have one icon
![image](https://github.com/user-attachments/assets/19ff56fe-68c8-402a-b4e0-86b40269b34f)

**Reply by OzGav** [reply]
You can change those icons in the settings https://music-assistant.io/ui/#player-list

**Reply by SN-git-stack** [reply]
Thank you, got it.
